### ISSUE TITLE
URL duchessefit.com not blocked yet

### ISSUE THREAD

**Issue by Alex67693** [hectorm/hblock#147]
This URL gives a fake virus/clean your phone warning.

Example: https://duchessefit.com/GXzUvWwVRKI_SNxOOWcfPItI87EQavKtfnW6FtK3gkA/?cid=w7d4b8cg4dngm39u2bilfjn8&sid=c119ff56-5479-4467-bbb6-73e3a4037167

Can be found on spam Google search results.

**Comment by hectorm** [comment]
I do not personally maintain a blocklist, hBlock is powered by several third party sources. If you want a domain to be blocked by default, you should submit it to one of these sources.

Anyway, the domain `duchessefit.com` is already covered by AdGuard and is therefore already blocked in hBlock.
